dotcom-rendering/src/web/components/Callout/FormField.tsx

```diff
@@ -152,7 +152,7 @@ export const FormField = ({
 						error={fieldError ? fieldError : undefined}
 						data-testid={`form-field-${formField.id}`}
 					>
-						{formField.options.map((option, index) => {
+						{formField.options.map((option) => {
```

Comment:
no need for index now we're using `option.value` as this should be unique

Reply:
Yeaa that's right. I'm going to revert this anyway as James has made other PR to fix it :)